omod/src/main/web/dashboardwidgets/programs/programs.controller.js

```diff
@@ -59,7 +59,9 @@ export default class ProgramsController {
             v: 'custom:(privileges:(name))'
         }).then((response) => {
             if (response && response.user && angular.isArray(response.user.privileges)) {
-                if (response.user.privileges.some( (p) => { return p.name === 'Task: coreapps.enrollInProgram'; })) {
+                if (response.user.roles.some( (p) => { return p.name === 'System Developer'; })) {
```

Comment:
Could we wrap this in a reusable function such that we do not have the role name `System Developer` in more than one place?

Reply:
Hi Daniel,
I had created a reusable function in the [dashboardwidgetscommons.service.js](omod/src/main/web/dashboardwidgets/dashboardwidgetscommons.service.js)